gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/src/main/java/io/gravitee/gateway/jupiter/handlers/api/SyncApiReactor.java

```diff
@@ -136,6 +152,8 @@ public Completable handle(final MutableRequestExecutionContext ctx) {
         prepareContextAttributes(ctx);
         prepareMetrics(ctx);
 
+        pendingRequests.incrementAndGet();
+
         return handleRequest(ctx);
```

Comment:
By doing this you can cover all the cases and avoid to call return `pendingRequests.decrementAndGet()` at multiple places:

```java
return handleRequest(ctx)
  .doFinally(() -> pendingRequests.decrementAndGet());
```